stone-payments/kong-plugin-template-transformer
# Bugfix/APIDC-2765: Template validation

## Description
- Fixes params validation that was not been called.

## How Has This Been Tested?
- `make install`
- `kong restart`
- Execute request to register an invalid template
- See error 400 with message.

## Checklist
- [x] VSTS build works.
